app/src/main/res/layout/project_layout.xml

```diff
@@ -13,16 +14,23 @@
   <androidx.recyclerview.widget.RecyclerView
     android:id="@+id/project_recycler_view"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:layout_above="@+id/project_action_buttons"
-    android:layout_below="@id/project_toolbar" />
+    android:layout_height="0dp"
+    android:layout_above="@+id/rewards_container"
+    android:layout_below="@id/project_toolbar"
+    app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
this is getting cut off by the footer button
<img src="https://user-images.githubusercontent.com/1289295/55744047-eda52100-5a01-11e9-9347-80b6a3c85846.png" width="330"/>